packages/vector_graphics/lib/src/listener.dart

```diff
@@ -243,6 +243,7 @@ class FlutterVectorGraphicsListener extends VectorGraphicsCodecListener {
 
   @override
   void onSize(double width, double height) {
+    _canvas.clipRect(ui.Offset.zero & ui.Size(width, height));
```

Comment:
This isn't necessary for this patch, but it's going to cause weird problems if we don't clip the drawing.